build/charts/flow-aggregator/templates/roles.yaml

```diff
@@ -26,19 +26,23 @@ rules:
   # RBAC to create / update / get flow-aggregator-ca ConfigMap
   - apiGroups: [""]
     resources: ["configmaps"]
-    verbs: ["create"]
```

Comment:
If possible, when it comes to list / watch / get / update for Secrets and ConfigMaps, let's stick to specific `resourceNames`

Reply:
Does this work with a shared informer created using the informerfactory?

Reply:
you may need separated filtered informers
hopefully we can have a single one per resource type, not sure if possible

Reply:
Yeah, I had previously a single informer and that was fine. I'd need one per resource in this case if I change the RBAC